packages/atlas/src/components/_navigation/NavItem/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './NavItem'
+export * from './NavItem.styles'
```

Comment:
I don't think we need to reexport styles, those generally shouldn't be used outside of the component implementation